Objective-C/CBLReplicator.mm

```diff
@@ -301,17 +300,13 @@ - (void) stop {
 }
 
 - (void) stopped {
-    CBL_LOCK(self) {
```

Comment:
Why was this lock removed?

Reply:
Because it's already called under the same lock.